GoogleForCreators/web-stories-wp
# Story progress bar not aligned with top of content

## Summary

Add media queries that cuts the top gap when it is too large. Or specific for the ratio between (435 / 874) and (490 / 874)

## Relevant Technical Choices

Use media queries to control the styling for thee class page-fullbleed-area.

## To-do

N/A

## User-facing changes

N/A

## Testing Instructions

- Go to the story page on Chrome.
- Under device, choose `responsive`, check the style for the ratio between  (435 / 874) and (490 / 874).
- Make sure the `progress bar + icons overlap with 9:16 content area` will not happen.

---

Fixes #5322
